packages/nextjs/app/builders/0x9dF97939e2aA22954CFddcD6a57008DA5d5F38Be/page.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from "next/image";
+import Link from "next/link";
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { faTwitter } from "@fortawesome/free-brands-svg-icons";
+import { faTelegram } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
I know Heroicons has a limited set of icons, which is probably why you're adding another library.
However, another PR is introducing react-icons, which includes multiple icon sets.

To keep things consistent, could you switch your icons to a react-icons option?

Thanks!